spec/helpers.lua

```diff
@@ -1304,6 +1733,8 @@ Expected CN:
 Got instead:
 %s
 ]])
+-- TODO: this seems broken, there seem to be no tests. A nicer assertion would be
+-- assert.certificate.has.cn("somename.com")
```

Comment:
If it's broken and no test is using it, I propose we remove it from the codebase instead.

Reply:
it makes sense to keep it around imo, but should be fixed at some point, hence the "todo" marker

Reply:
```
assert.certificate(cert).has.cn("somename.com")
```